src/fastga_he/powertrain_builder/powertrain.py

```diff
@@ -498,7 +566,7 @@ def _get_connections(self):
                 target_id == "fastga_he.pt_component.dc_bus"
                 or target_id == "fastga_he.pt_component.dc_splitter"
             ):
-                self._sspc_list[source_name] = True
+                pt_cache["_sspc_list"][source_name] = True
```

Comment:
Why use the cache here ? It's quicker and more coherent to call the attributes. The reason why is that we have already called the generation of component list which itself checked and imported everything from the cache already. So it should be safe and same line 591